### DISCUSSION TITLE
Externally connect to Kafka connect

### DISCUSSION THREAD

**Discussion by itaydj** [strimzi/strimzi-kafka-operator#5389] [Q&A] [answered]
Hi,
I checked the (amazing) Documentation but couldnt find a way to expose my Kafka connect instance.
I want to be able to connect to it from an external tool such as Conducktor.
Is that supported in any way? (ingress, Nodeport)

Thanks

**Comment by scholzj** [comment, marked as answer]
There is no way to do it through the operator. But you can do it your self and for example create your own Ingress or node port service and use it. Just keep in mind there is right now no security. So you should be careful about it.

**Reply by itaydj** [reply]
Got it,
Thanks alot.
